ThunderCloud/CollectionItemViewCell.swift

```diff
@@ -156,7 +222,49 @@ open class CollectionItemViewCell: UICollectionViewCell {
             return [imageView?.accessibilityLabel, titleAccessibilityLabel].compactMap({ $0 }).joined(separator: ",")
         }
         set {
-            
         }
     }
 }
+
+// MARK: - Extensions
+
+extension CollectionCellDisplayable {
+
+    /// Top label text
+    var title: String {
+        return itemTitle?.trimmingCharacters(in: .whitespacesAndNewlines) ?? ""
+    }
+    
+    /// Bottom label text
+    var expiryDateString: String {
+        return expirableAchievement?.expiryDateString ?? ""
+    }
+}
+
+extension UIEdgeInsets {
```

Comment:
Been thinking of adding an extension like this into ThunderBasics recently as seem to use it everywhere! Could you create an issue on that Repo to do so? :D

Reply:
Off develop yeah if you do come back to this :)